doc/introduction/index.rst

```diff
@@ -3,383 +3,363 @@
+pins to be associated with a guest, MSI transforms interrupts into messages that
+are more easily virtualized, scaling to thousands of individual interrupts. MSI
+has been available since PCI version 2.2 and is also available in PCI Express
+(PCIe).  MSI is ideal for I/O virtualization, as it allows isolation of
+interrupt sources (as opposed to physical pins that must be multiplexed or
+routed through software).
 
-In this Hybrid real-time (RT) scenario, a pre-launched RTVM is started by the
-hypervisor. The Service VM runs a post-launched User VM that runs non-safety or
-non-real-time tasks.
+Hypervisor Support for Device Passthrough
+=========================================
 
-.. figure:: images/ACRN-Hybrid-RT.png
-   :width: 600px
-   :align: center
-   :name: ACRN-Hybrid-RT
+By using the latest virtualization-enhanced processor architectures, hypervisors
+and virtualization solutions can support device passthrough (using VT-d),
+including Xen, KVM, and ACRN hypervisor.  In most cases, the guest operating
+system in the User VM, must be compiled to support passthrough by using kernel
+build-time options.
+
+
+Static Configuration Based on Scenarios
+***************************************
+
+Scenarios are a way to describe the system configuration settings of the ACRN
+hypervisor, VMs, and resources they have access to that meet your specific
+application's needs such as compute, memory, storage, graphics, networking, and
+other devices.  Scenario configurations are stored in an XML format file and
+edited using the ACRN configurator tool.
+
+Following a general embedded-system programming model, the ACRN hypervisor is
+designed to be statically customized at build time per hardware and scenario,
+rather than providing one binary for all scenarios.  Dynamic configuration
+parsing is not used in the ACRN hypervisor for these reasons:
+
+* **Reduce complexity**. ACRN is a lightweight reference hypervisor, built for
+  embedded IoT. As new platforms for embedded systems are rapidly introduced,
+  support for one binary could require more and more complexity in the
+  hypervisor, which is something we strive to avoid.
+* **Maintain small footprint**. Implementing dynamic parsing introduces hundreds or
+  thousands of lines of code. Avoiding dynamic parsing helps keep the
+  hypervisor's Lines of Code (LOC) in a desirable range (less than 40K).
+* **Improve boot time**. Dynamic parsing at runtime increases the boot time. Using a
+  static build-time configuration and not dynamic parsing helps improve the boot
+  time of the hypervisor.
+
+The scenario XML file together with a target HW board XML file are used to build
+the ACRN hypervisor image tailored to your HW and application needs. The ACRN
+project provides a board inspector tool to automatically create the board XML
+file by inspecting the target hardware. ACRN also provides a
+:ref:`configurator tool <acrn_configuration_tool>`
```

Comment:
```suggestion
:ref:`configuration tool <acrn_configuration_tool>`
```

Reply:
we're settled in on calling it by the tool's name (configurator) rather than its purpose (configuration)